playbooks/byo/openshift-cluster/upgrades/docker/docker_upgrade.yml

```diff
@@ -23,11 +23,23 @@
   serial: 1
   any_errors_fatal: true
   tasks:
-  - name: Prepare for Node draining
-    command: >
-      {{ openshift.common.client_binary }} adm manage-node {{ openshift.node.nodename }} --schedulable=false
+  - name: Load lib_openshift modules
+    include_role:
+      name: lib_openshift
+
+  - name: Mark node unschedulable
+    oadm_manage_node:
+      node: "{{ openshift.node.nodename | lower }}"
+      schedulable: False
     delegate_to: "{{ groups.oo_first_master.0 }}"
-    when: l_docker_upgrade is defined and l_docker_upgrade | bool and inventory_hostname in groups.oo_nodes_to_upgrade
+    retries: 10
+    delay: 5
```

Comment:
@mtnbikenc,

https://bugzilla.redhat.com/show_bug.cgi?id=1279303

I think modules can be more resilient.  That is always the goal.  Then the discussion needs to be how long do we wait? How many times do we retry? This is what @mwoodson was mentioning. I'm definitely willing to discuss that as I think we could greatly benefit from guaranteeing state before proceeding.